src/Modules/trailsData.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
+
+const apiUrl = 'http://localhost:3000/v1/'
+
+const getTrails = async () => {
+  try {
+    let response = await axios.get(apiUrl + 'trails')
+    return response.data.trails
+  } catch(error) {
+    return {
+      error: error.response.data.error
```

Comment:
are you sure its .error if you have .error_message in the actual function in component, or does the response look `error.response.data.error.error_message` or `error.response.data.error_message` if second one then remove the last .error